sdk/storage/azure-storage-blob/tests/perfstress_tests/download.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.perfstress_tests import get_random_bytes, WriteStream
+
+from ._test_base import _ContainerTest
+
+
+class DownloadTest(_ContainerTest):
```

Comment:
Should this extend `_BlobTest` instead of `_ContainerTest`?

Reply:
As mentioned above when discussing the upload/download resource patter - the `_BlobTest` base represents a unique endpoint to that instance - whereas for download we don't need a unique resource.